task4-1/binaryAdditionTest.c

```diff
@@ -0,0 +1,113 @@
+#include "binaryAddition.h"
+#include <stdbool.h>
+#include <stdio.h>
+
+bool testDecimalToBinary() {
+    int num = -15;
+    int bin[INT_SIZE] = {};
```

Comment:
А Си уже умеет пустые инициализаторы? Так более канонично:
```suggestion
    int bin[INT_SIZE] = {0};
```
Справедливости ради, функции без аргументов должны объявляться как `bool testDecimalToBinary(void)`, это ж не C++. Но, опять же справедливости ради, за последними стандартами Си я не слежу.